components/availability/src/Availability.svelte

```diff
@@ -317,34 +329,26 @@
               class="slot {slot.selectionStatus} {slot.availability}"
               data-start-time={new Date(slot.start_time).toLocaleString()}
               data-end-time={new Date(slot.end_time).toLocaleString()}
-              on:mouseover={() =>
-                console.log(
-                  "TODO: temp; ",
-                  new Date(slot.start_time).toLocaleString(),
-                )}
+              disabled={slot.availability === "busy"}
```

Comment:
Note for our future selves: we'll eventually want a property like `allow_partial_bookings` that makes this slightly more conditional.